src/components/pages/users/cards/roles.tsx

```diff
@@ -0,0 +1,52 @@
+import type {FunctionComponent} from 'react'
+import {AccordionItem, AccordionButton, AccordionPanel} from '@reach/accordion'
+
+import '@reach/accordion/styles.css'
+import {Card} from '@/components/lib'
+import {IRole} from '@/types'
+
+const RoleCard: FunctionComponent<IRole> = ({
+  name,
+  canTriageRequests,
+  canEditSettings,
+  canCreateUsers,
+  canCreateGroups,
+  canEditRoles,
+  canManageInfrastructure,
+  canUploadData
+}) => {
+  const permissions = [
```

Comment:
Extract permissions out of the component. Maybe this could be in a JSON somewhere?